example-rule-library/README.md

```diff
@@ -0,0 +1,68 @@
+<!--
+ Copyright 2018-2021 Crown Copyright
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+# <img src="../logos/logo.svg" width="180">
+
+## A Tool for Complex and Scalable Data Access Policy Enforcement
+
+# Example Rule library
+Contains the Rules and supporting code used in the example.
+Each of these users is defined by the ExampleUser which is a specialisation of the [Palisade-common](https://github.com/gchq/Palisade-common/README.md) Rule interface.
```

Comment:
I think we need to expand this to also talk about the setting of the spring boot config (e.g. pre pop stuff)